vocabularies/Common.xml

```diff
@@ -569,6 +569,15 @@ The minimum structure is
         </Property>
       </ComplexType>
 
+      <Term Name="Composition" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="NavigationProperty">
```

Comment:
```suggestion
      <Term Name="Composition" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="NavigationProperty">
        <Annotation Term="Core.AppliesViaContainer"/>
```
Do we want to add this?

Reply:
No, I wanted to directly annotate the navigation properties, see the example.